packages/generators/templates/docs/{{name}}.stories.{{mdx}}

```diff
@@ -47,13 +47,6 @@ The {{name}} will... _Add a brief description of the design and usage of the com
   Is there any related resource (internal or external) to this component?
 -->
 
-## Platform considerations
```

Comment:
This and the Notes section are in the Implement tab and therefore, taken out of the stories template.